index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+<meta charset="UTF-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Hello, Welome!</title>
+<link rel="stylesheet" href="style.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+</head>
+
+<body>
+   <navbar class="navbar-fixed-top">
+<p>Welcome! <img src="/home/edwin/Portfolio-webpage/Images/Capacity.jpg"> <img src="/home/edwin/Portfolio-webpage/Images/Icon - Menu.png"> <img src="/home/edwin/Portfolio-webpage/Images/Wifi.png"> </p>
+   </navbar>
+    <header class="heading"> 
+
+    <h1 class="box"> Hey there. I’m Edwin! </h1> 
+    <h2 class="box"> I’m a software developer :)</h2> 
+    <article class="section">
+       <p> I can help you build a product, feature or website. Have a look at some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.</p>
+     </article>
+     <section>
+        <p> Please reach out!
+<a href="https://web.facebook.com/?_rdc=1&_rdr" class=" fa fa-facebook"> Facebook</a>
+<a href="https://twitter.com/login" class="fa fa-twitter"> Twitter</a> </p>
```

Comment:
- Kindly, list all social icons as the Figma design, and use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).
Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href. 👌